pootle/apps/accounts/management/commands/merge_user.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import accounts
+
+from . import UserCommand
+
+
+class Command(UserCommand):
+    args = "user other_user"
+    help = "Merge user to other_user"
+
+    def handle(self, *args, **kwargs):
+        self.check_args(*args)
```

Comment:
Since this is being called in every command inheriting from `UserCommand`, wouldn't it make sense to leave this responsibility to the base class, and call `super()` from actual commands?